cmd/vela-s3-cache/main.go

```diff
@@ -224,13 +226,14 @@ func run(c *cli.Context) (err error) {
 	}
 
 	logrus.WithFields(logrus.Fields{
-		"code": "https://github.com/go-vela/vela-s3-cache",
-		"docs": "https://go-vela.github.io/docs/plugins/registry/s3-cache",
-		"time": time.Now(),
+		"code":     "https://github.com/go-vela/vela-s3-cache",
+		"docs":     "https://go-vela.github.io/docs/plugins/registry/s3-cache",
+		"registry": "https://hub.docker.com/r/target/vela-s3-cache",
+		"version":  version.New().Canonical,
```

Comment:
sorry - must have been blind and didn't see the other version output. i was hoping to just see a `v1.0.0` somewhere. i will remove to stay consistent. thanks for catching that.

Reply:
No problem! But FWIW, I'm not opposed to what you had here 👍 

And as far as consistency goes, we could always update the other plugins to do it the way you have it now?

However, for the sake of transparency, I do like the full dump of version information that includes the `metadata`.

I say this because (I hope) at some point in the future I could see us creating plugins for different operating systems and architectures.

So having that metadata output directly in the logs for the plugin is a "nice to have" in those scenarios.

Especially when troubleshooting issues in relation to compatibility.

Reply:
that's fair. i think i'll follow suit for now. perhaps the verbose dump could be used for `debug` log level and the short and sweet for `info`? but i'm good tackling that separately.